source/configuration_manual/howto/antispam_with_sieve.rst

```diff
@@ -12,25 +12,22 @@ Caveats and possible pitfalls
 
```

Comment:
While you're fixing things. This file begins with a weird `. _howto-antispam_with_imapsieve` line.